scripts/deploy.sh

```diff
@@ -105,6 +115,21 @@ EOM
 
 fi
 
+if [[ "$CANISTER" == "" ]] || [[ "$CANISTER" == "signing" ]]; then
+
+echo
+echo == Install signing.
+echo
+
+canister install signing --mode="$MODE" --argument "$(cat << EOM
+(record {
+  owners = vec { $OWNERS; principal "$(canister id deposits)" };
+  key_id = "$SIGNING_KEY_ID";
+})
```

Comment:
Consider using controllers here instead of separate owners. But that complicates the deployment script